instrumentation/src/main/java/io/opentelemetry/android/OpenTelemetryRumBuilder.java

```diff
@@ -308,7 +314,34 @@ private SdkTracerProvider buildTracerProvider(SessionId sessionId, Application a
     private SpanExporter buildSpanExporter() {
         // TODO: Default to otlp...but how can we make endpoint and auth mandatory?
         SpanExporter defaultExporter = LoggingSpanExporter.create();
-        return spanExporterCustomizer.apply(defaultExporter);
+        SpanExporter spanExporter;
+        DiskBufferingConfiguration diskBufferingConfiguration =
+                config.getDiskBufferingConfiguration();
+        if (diskBufferingConfiguration.isEnabled()) {
+            try {
+                spanExporter = createDiskExporter(defaultExporter, diskBufferingConfiguration);
+            } catch (IOException e) {
+                spanExporter = defaultExporter;
```

Comment:
Probably worth logging this exception so that there's a chance of seeing the problem. It's only once at startup time, so hopefully not considered too verbose.

Reply:
Sounds good, I've added a log there.